faq/faqs-business-logic.md

```diff
@@ -0,0 +1,79 @@
+
+If not stated otherwise, the content is applicable both to {{site.bl-data}} and {{site.bl-func}}.
+
+* [Can I determine which user account is making the request?](#can-i-determine-which-user-account-is-making-the-request)
+* [How do I get the body of the incoming request?](#how-do-i-get-the-body-of-the-incoming-request)
+* [How do I get the result status of the request?](#how-do-i-get-the-result-status-of-the-request)
+* [Can I apply filtering criteria before returning data?](#how-to-apply-filtering-criteria-before-returning-data)
+* [How do I determine the Id of the requested item?](#how-do-i-determine-the-id-of-the-requested-item)
+* [Can I send a push notification when an item is created/updated/deleted?](#can-i-send-a-push-notification-when-an-item-is-createdupdateddeleted)
+
+## Can I determine which user account is making the request?
+
+You can get information about the account but only if the request is made by an authenticated user from the information contained in the `request.principal.data` object of the requests parameters.
+
+More information is available in [this article]({% slug bl-determine-user-account %}).
+
+## How do I get the body of the incoming request?
+
+You can get it from the request parameters.
+
+More information is available in [this article]({% slug bl-get-request-body %}).
+
+## How do I get the result status of the request?
+
+In the *after* events of the {{site.bl-data}} you can get information about the completion status of the data operation from the `response` parameters.
+
+More information is available in [this article]({% slug bl-get-request-result %}).
+
+## How to apply filtering criteria before returning data?
+
+**Applicable only to {{site.bl-data}}**. You can do this in the {{site.bl-data}} `before*` event handlers.
+
+More information is available in [this article]({% slug bl-apply-filter %}).
+
+## How do I determine the Id of the requested item?
+
+**Applicable only to {{site.bl-data}}**. You can get the `Id` of the requested item from the request parameters.
+
+More information is available in [this article]({% slug bl-id-requested-item %}).
+
+## Can I send a push notification when an item is created/updated/deleted?
+
+Yes. Use the [Cloud Code SDK](http://docs.telerik.com/platform/backend-services/javascript/apireference/CloudCode/Everlive/Everlive) in the appropriate `before*` or `after*` event handler to send a push notification. Read more in [Sending Push Notifications](http://docs.telerik.com/platform/backend-services/javascript/push-notifications/send-and-target/push-send).
+
+## See Also
+
+* [Getting Started with {{site.bl-data}}](http://docs.telerik.com/platform/backend-services/javascript/server-side-logic/cloud-code/cloud-code-for-data/cloud-code-for-data-getting-started)
```

Comment:
Consider creating a site variable to hold the domain or going with relative links (you can talk their format over with Evgeni).